cirrus/server/tests/test_telemetry.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+import pytest
+
+from cirrus.experiment_recipes import RecipeType
+from cirrus.main import (
+    FeatureRequest,
+    app,
+    compute_features,
+    initialize_glean,
+    record_metrics,
+)
+from cirrus.sdk import SDK
+from glean.testing import ErrorType
+
+
+# This method is shared between two tests
+# One is a unit test, with manually generate enrollment events
+# The other is an integration test, with
```

Comment:
"with...." 😄

Reply:
I think you can just delete this comment altogether

Reply:
idk if it's very "pythony" of me, but if this method is shared by two tests in two different files, maybe put it in a separate test helper file?

Reply:
So I was thinking the same thing and I ultimately chose to leave it in here because its assertions are specifically related to the input of the tests

Reply:
Am I blind? I don't see where this is used in the tests

Reply:
It's used in this method: https://github.com/mozilla/experimenter/pull/9140/files#diff-4dfd00a1e8b6710be16bae09b260df5f894f346d70ebf70a658376c2a0441772R76

Reference here: https://mozilla.github.io/glean/python/glean/metrics/ping.html#glean.metrics.ping.PingType.test_before_next_submit